source/includes/_egress.md

```diff
@@ -107,23 +105,23 @@ curl https://api.onramp.ltd/rpc/create_egress_invoice                     \
-payment_ack_url   | String          | Merchant callback endpoint to confirm egress transaction. It should be a complete, well formed, url.
-user_redirect_url | String          | Where to redirect the user after the egress has been confirmed. It should be a complete, well formed, url.
-timeout_in_sec    | Integer         | When to expire the link if unused.
-offer_skin        | Egress Skin     | Specify how the offer should be displayed to the user.
-billing_details   | Billing Details | User billing details. Please, notice how **not** all parameters inside this json object are required except `merchant_customer_id`.
+Field             |   Type          | Description                                                                                                                          | Required 
+----------------- | --------------- | ------------------------------------------------------------------------------------------------------------------------------------ | --------
+fiat_amount       | Integer         | Eur amount to be paid denominated in cents.                                                                                          | Yes
+fiat_currency     | String          | `"EUR"` or `"USD"`, depending what is available on the merchants account settings.                                                   | Yes 
+payment_ack_url   | String          | Merchant callback endpoint to confirm egress transaction. It should be a complete, well formed, url.                                 | Yes
+user_redirect_url | String          | Where to redirect the user after the egress has been confirmed. It should be a complete, well formed, url.                           | Yes
+timeout_in_sec    | Integer         | When to expire the link if unused.                                                                                                   | Yes
+offer_skin        | Egress Skin     | Specify how the offer should be displayed to the user.                                                                               | Yes
+billing_details   | Billing Details | User billing details. Please, notice how **not** all parameters inside this json object are required except `merchant_customer_id`.  | Yes
 
 ### Egress Skin
 
-Field             |   Type      | Description
------------------ | ----------- | -----------
-title             | string      | Short string containing merchant's or redemption's name.
-image             | url         | image to stylized the offer.
-description       | string      | A text explaining what the user is redeeming.
+Field             |   Type      | Description                                               | Required
+----------------- | ----------- | --------------------------------------------------------- | --------
+title             | string      | Short string containing merchant's or redemption's name.  | Yes
+image             | url         | image to stylized the offer.                              | Yes
```

Comment:
Here we are already using `url` instead of `string`, but somewhere else we don't (I know this is not from the PR but from a previous one). Also another inconsistency, probably it would look better if we stick to a consistent capitalization for both the `Field` field and the `Type` field.

Reply:
Made those changes too.